lib/ansible/executor/task_executor.py

```diff
@@ -329,7 +328,6 @@ def _squash_items(self, items, loop_var, variables):
             # optimizing it here, the templatable string might use template vars
             # that aren't available until later (it could even use vars from the
             # with_items loop) so don't make the templated string permanent yet.
-            templar = Templar(loader=self._loader, shared_loader_obj=self._shared_loader_obj, variables=variables)
```

Comment:
This may not be backwards compatible.  It looks like here we can pass in `variables`.  If `variables` is passed into `_squash_items` then `templar` doesn't have those explicitly passed variables.

It looks like we do actually pass `variables` into this method.